clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/cache/backend/ObjectCacheConfig.java

```diff
@@ -1,11 +1,13 @@
 package com.linkedin.davinci.store.cache.backend;
 
+import com.linkedin.davinci.client.CacheValueTransformer;
 import java.util.Optional;
 
 
 public class ObjectCacheConfig {
   private Optional<Long> maxCacheSize = Optional.empty();
```

Comment:
We plan to use global DVC config to specify the shared object cache size for all the stores.